shell/plugins/plugin.js

```diff
@@ -18,5 +20,57 @@ export default async(context) => {
     }
   });
 
-  await allHash(hash);
+  // Discover all of the UI Plugins CRs
+
+  let loadPlugins = true;
+
+  // Provide a mechanism to load the UI without the plugins loaded - in case there is a problem
+  if (context.route?.path === '/safeMode') {
```

Comment:
Do we want to skip out ALL plugin stuff, including `context.$plugin.loadAsync` above?

Reply:
Fixed